packages/providers/anthropic/src/llm.ts

```diff
@@ -30,6 +30,8 @@ import { ToolCallLLM } from "@llamaindex/core/llms";
 import { extractText } from "@llamaindex/core/utils";
 import { getEnv } from "@llamaindex/env";
 import { isDeepEqual } from "remeda";
+import { z } from "zod";
+import { zodToJsonSchema } from "zod-to-json-schema";
```

Comment:
use zod v4, it's to json schema built-in, you can refer to my another PR https://github.com/anthropics/dxt/pull/39/files#diff-00e8b7e9ce9d95fa021ecce03324e02fe9b946eaeaf0c2f779c0c5d9278ef051